DenisZackharov/teacherss
# CRUD for field of study

### Summary
Same as: ⚠️Parent: #109, with little updates

### How it works

<img width="1409" alt="image" src="https://user-images.githubusercontent.com/78980031/231293420-f0cbca78-e1d3-4989-91f8-bde7a71c56f9.png">

### Test plan

List of steps to manually test introduced functionality:

1. Login as director with credentials: `director@example.com` `123456`
2. Click to organization name `School 13`
3. Click`Field of studies`
4. You can create new field of study, edit and destroy existed

1. Login as teacher with credentials: `teacher@example.com` `123456`
2. Click to organization name `School 13`
3. Click `Field of studies`
4. You can see existed field of studies

# Checklist:


### Deploy notes
None.
